### PR TITLE
back to html5 color picker

### PR DESCRIPTION
**Pull request by dmossakowski**
removing Huebe color picker as it didn't work well on mobile

### CODE REVIEW

File: views/gym-routes.html [hunk 1]
```diff
@@ -409,9 +410,27 @@
                     //console.log( 'color changed to: ' + color )
                     modalBoxDiv.style.border="5px solid "+color;
                     hueb.close();
-        })
-
+        })*/
 
+        
+        function colorChanged(tp){
+          console.log('color changed');
```
Comment (scorreia) [inline views/gym-routes.html:417]: you can probably remove the log now

File: views/gym-routes.html [hunk 2]
```diff
@@ -389,12 +390,12 @@
...
             
             // this is a background on edit popup input field
-            document.getElementById('color1').style.backgroundColor = currentRow.color1;
+            //document.getElementById('color1').style.backgroundColor = currentRow.color1;
             document.getElementById('hidden_message').textContent='';
         }
 
         // use selector string to initialize on single element
-        var hueb = new Huebee( '.color-input', {
+        /*var hueb = new Huebee( '.color-input', {
```
Comment (scorreia) [inline views/gym-routes.html:398]: can we delete this code in comment?

File: views/skala3ma-layout.html [hunk 3]
```diff
@@ -102,6 +102,8 @@
 
     <!-- script for custom color picker-->
     <script src="/public/js/huebee.pkgd.min.js"></script>
```
Comment (scorreia) [inline views/skala3ma-layout.html:104]: did you completely remove huebee? or is this line still required?

**Review by scorreia — DISMISSED** [review]
minor changes suggested regarding commented code, but it's working. So we can merge.

File: views/skala3ma-layout.html (hunk 3 above)
Comment (dmossakowski) [reply views/skala3ma-layout.html]: done in latest commit

File: views/gym-routes.html (hunk 2 above)
Comment (dmossakowski) [reply views/gym-routes.html]: done

File: views/gym-routes.html (hunk 1 above)
Comment (dmossakowski) [reply views/gym-routes.html]: done

**Pull request merged**